rangefilter/static/rangefilter/iife.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+  'use strict';
+  django.jQuery(".admindatefilter").each(
+    function(){
+      var form_id = django.jQuery(this).find("form").attr('id').slice(0,-5);
```

Comment:
Why `.slice(0,-5)`?

Reply:
To drop "-form" from the string and get just the variable part.